Tests/scripts/slack_notifier.py

```diff
@@ -202,10 +223,40 @@ def slack_notifier(build_url, slack_token, env_results_file_name, container):
         )
 
 
+def sdk_slack_notifier(build_url, slack_token, container):
```

Comment:
very similar to `slack_notifier` function.
add an argument for `is_sdk_nightly` there and change `env_results_file_name` to optional

Reply:
Done